blackbox/docs/best_practice/full_restart_upgrade.txt

```diff
@@ -0,0 +1,171 @@
+.. highlight:: sh
+
+.. _full_restart_upgrade:
+
+====================
+Full Restart Upgrade
+====================
+
+The preferred way to upgrade a running CrateDB cluster is via the
+:ref:`rolling_upgrade` method. But this is not always an option (for example,
+when you are upgrading from one minor version to the next).
```

Comment:
nope. minor to minor requires restart. it's only patch to patch that you can do a rolling upgrade